src/sidebar/services/test/load-annotations-test.js

```diff
@@ -0,0 +1,268 @@
+import EventEmitter from 'tiny-emitter';
+
+import loadAnnotationsService, { $imports } from '../load-annotations';
+
+let searchClients;
+let longRunningSearchClient = false;
+class FakeSearchClient extends EventEmitter {
+  constructor(searchFn, opts) {
+    super();
+
+    assert.ok(searchFn);
+    searchClients.push(this);
+    this.cancel = sinon.stub();
+    this.incremental = !!opts.incremental;
+
+    this.get = sinon.spy(query => {
+      assert.ok(query.uri);
+
+      for (let i = 0; i < query.uri.length; i++) {
+        const uri = query.uri[i];
+        this.emit('results', [{ id: uri + '123', group: '__world__' }]);
+        this.emit('results', [{ id: uri + '456', group: 'private-group' }]);
+      }
+      if (!longRunningSearchClient) {
+        this.emit('end');
+      }
+    });
+  }
+}
+
+describe('loadAnnotationsService', () => {
+  let fakeStore;
+  let fakeApi;
+  let fakeAnnotationMapper;
+  let fakeStreamer;
+  let fakeStreamFilter;
+
+  let fakeUris;
+  let fakeGroupId;
```

Comment:
`fakeUris` and `fakeGroupId` are never mutated AFAICS, so they could just be constants.

Reply:
Works in the case of `fakeGroupId` but `fakeUri`s _is_ mutated in a test.